utils.py

```diff
@@ -15,9 +13,48 @@ def read_lines(file_name, is_strip=True):
         lines = [s.strip() for s in lines]
     return lines
 
-def translate_to_chars(ascii_vals):
```

Comment:
only real risk, 2019 day 25 is broken now, but easily fixed (same method, new name)